toponetx/classes/cell_complex.py

```diff
@@ -870,6 +870,114 @@ def get_filtration(self, name: str) -> dict[Union[Hashable, tuple], Any]:
                 d.update(i)
         return d
 
+    def set_node_attributes(
+        self,
+        values: Union[dict[Hashable, dict], dict[Hashable, Any]],
+        name: Optional[str],
+    ):
+        """Set node attributes.
+
+        Parameters
+        ----------
+        values :  dict
+            either contains node -> value (if `name` is specified)
+            or nested dict with node -> (attribute -> value) (if `name == None`)
+        name : str, optional
+
+        Returns
+        -------
+        None.
```

Comment:
Functions that don't return anything don't have a `Returns` section at all as per NumPy's documentation style.